trainer.py

```diff
@@ -91,24 +76,21 @@ def train_transductive(
     model.updateZ(logits_lp)
 
     pred = model.lp_score(logits_lp, ei, et)
-
     y = torch.cat(
         [logits_lp.new_ones(pei.size(-1)), logits_lp.new_zeros(nei.size(-1))], dim=0
     )
 
-    loss_ = model.lp_loss(pred, y)
-    loss = loss_ if loss is None else loss + loss_
+    loss = model.lp_loss(pred, y)
     lp_auc, lp_ap = model.lp_test(pred, y)
+
     if ((mode == "test")) and cal_mrr_score:
         model.lp_log_ranks(logits_lp, pei, pet, data.edge_index, data.edge_type)
-
-    total_loss += loss * num_graphs
```

Comment:
The total_loss variable should still be there right?

Reply:
We removed the loop "for data in loader"

Total_loss is always the same as loss